acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/timeouts/ConnectionTimeoutAcceptanceTest.java

```diff
@@ -81,8 +81,7 @@ public void submittingTransactionReturnsAGatewayTimeoutError() {
     final SignerResponse<JsonRpcErrorResponse> signerResponse =
         ethSigner.transactions().submitExceptional(transaction);
     assertThat(signerResponse.status()).isEqualTo(GATEWAY_TIMEOUT);
-    assertThat(signerResponse.jsonRpc().getError())
-        .isEqualTo(CONNECTION_TO_DOWNSTREAM_NODE_TIMED_OUT);
+    assertThat(signerResponse.jsonRpc().getError()).isEqualTo(FAILED_TO_CONNECT_TO_DOWNSTREAM_NODE);
```

Comment:
Think these tests are misnamed, this error happen if we can't connect to the downstream web3 provider?

Reply:
Renamed, and added a test to prove PassThrough behaves the same way